holoviews/plotting/bokeh/chart.py

```diff
@@ -264,6 +264,9 @@ class SpikesPlot(PathPlot):
     position = param.Number(default=0., doc="""
       The position of the lower end of each spike.""")
 
+    show_legend = param.Boolean(default=True, doc="""
```

Comment:
This is a parameter available for the matplotlib backend IIRC. In which case, it is good to see this support added to the Bokeh backend.